src/doom/r_data.c

```diff
@@ -167,6 +167,8 @@ fixed_t*	spritetopoffset;
 
 lighttable_t	*colormaps;
 
+// [crispy] lookup table of color shades
+byte color_shades[NUMSHADES * 256];
```

Comment:
Even though other tables are initialized here, the `color_shades[]` array is really only used in `am_map.c`, so I don't see much sense to move its initialization code into the more general `r_data.c`. I'd prefer if this was moved into `am_map.c` and made `static` - and its initialization moved to the end of `AM_LevelInit()` with a simple plausibility check to avoid re-initialization.